tests/kuttl/tests/basic-deployment/02-cleanup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+- apiVersion: core.openstack.org/v1beta1
+  kind: OpenStackControlPlane
+  name: openstack
+  namespace: $NAMESPACE
```

Comment:
I had to remove all of these `namespace: $NAMESPACE` lines in all the `02-cleanup.yaml` and `02-errors-cleanup.yaml` files in both test dirs, otherwise these steps did not actually clean-up the `OpenStackControlPlane`.